src/explorers/ethereum/etherscan.js

```diff
@@ -1,17 +1,17 @@
-import { request } from '../services';
-import { BLOCKCHAINS, CONFIG, SUB_STEPS, TRANSACTION_APIS } from '../constants';
-import { TransactionData, VerifierError } from '../models';
-import { stripHashPrefix } from './utils/stripHashPrefix';
-import { getText } from '../domain/i18n/useCases';
-import { TRANSACTIONS_APIS_URLS } from '../constants/api';
+import { request } from '../../services';
+import { BLOCKCHAINS, CONFIG, SUB_STEPS, TRANSACTION_APIS } from '../../constants';
+import { TransactionData, VerifierError } from '../../models';
+import { stripHashPrefix } from '../utils/stripHashPrefix';
+import { getText } from '../../domain/i18n/useCases';
+import { TRANSACTIONS_APIS_URLS } from '../../constants/api';
 
 export function getEtherScanFetcher (transactionId, chain) {
   const action = '&action=eth_getTransactionByHash&txhash=';
   let etherScanUrl;
   if (chain === BLOCKCHAINS.ethmain.code) {
     etherScanUrl = TRANSACTIONS_APIS_URLS[TRANSACTION_APIS.Etherscan].main + action + transactionId;
   } else {
-    etherScanUrl = TRANSACTIONS_APIS_URLS[TRANSACTION_APIS.Etherscan].ropsten + action + transactionId;
+    etherScanUrl = TRANSACTIONS_APIS_URLS[TRANSACTION_APIS.Etherscan].test + action + transactionId;
```

Comment:
why not do the same thing with ETH APIs here?

Reply:
This is coming in the next PR ;)